swiftly/client/standardclient.py

```diff
@@ -514,19 +516,19 @@ def request(self, method, path, contents, headers, decode_json=False,
                         method, conn_path + path, contents, titled_headers)
                 else:
                     conn.putrequest(method, conn_path + path)
-                    content_length = None
```

Comment:
This change seems wrong, making content_length = 0 by default instead of None and then using 0 to decide whether to add the transfer-encoding: chunked header. If there's a content-length: 0 header, the transfer-encoding: chunked header will also be sent. Does Python 3 have trouble with Nones in certain cases?